BUILD.md

```diff
@@ -22,11 +22,19 @@ To compile Cemu, a recent enough compiler and STL with C++20 support is required
 
 For ubuntu and most derivatives:
 
-1) `sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
-2) Run `git clone --recursive https://github.com/cemu-project/Cemu`
-3) `cd Cemu`
-4) `mkdir build && cd build`
-5) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja`
-6) `ninja`
-
-Build instructions for other distributions will be added in the future!
+
+`sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
+
+For Arch and most derivatives:
+
+`sudo pacman -S --needed gtk3 libsecret libgcrypt systemd-libs freeglut nasm git zlib vcpkg zip unzip`
+
+For Fedora and most derivatives:
+
+`sudo dnf in -y gtk3-devel libsecret-devel libgcrypt-devel systemd-devel freeglut-devel nasm cmake vcpkg perl-core zlib-devel cubeb-devel`
+
+1) Run `git clone --recursive https://github.com/cemu-project/Cemu`
+2) `cd Cemu`
+3) `mkdir build && cd build`
+4) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja`
+5) `ninja`
```

Comment:
Pardon my interjection, but using `1.` (instead of `1)`) is the correct way to create numbered lists in Markdown.

Reply:
Just left a comment as I was poking around with the code/docs to see if there's anything I could provide myself, apologies in advance if a whole review comes across as a bit aggressive.

Reply:
That package install list for Fedora won't install ```llvm``` or ```clang```
Also why can't you change ```dnf in``` to ```dnf install``` ? 
I get that the former will work, but using shorthand for official documentation doesn't seem like the best approach.

Reply:
i am leaving it according to how it was by exzap for now.

Reply:
> That package install list for Fedora won't install `llvm` or `clang`
> Also why can't you change `dnf in` to `dnf install` ?
> I get that the former will work, but using shorthand for official documentation doesn't seem like the best approach.

sudo dnf in is just a better approach fedora users are used to it plus people tend to copy and paste. 

Clang came pre installed on 36/37 for me and i didn’t need llvm its why it didn’t get if you can show it does then i can add it.

Reply:
from ```man dnf```
```
       Available commands:

       • install
```
The command is called ```install``` and it's simply not a better approach to use a short-hand name for that in official documentation.  Also clang is not installed by default in any build I've seen. And it's definitely  not included in the 'Fedora Workstation'' environmental group. Yes you need to install ```llvm``` as well if you're using ```clang```. I can show you....just need a few minutes....

Reply:
> from `man dnf`
> 
> ```
>        Available commands:
> 
>        • install
> ```
> 
> The command is called `install` and it's simply not a better approach to use a short-hand name for that in official documentation. Also `clang` is not installed by default in any build I've seen. And it's definitely not included in the 'Fedora Workstation' environmental group. Also, you shouldn't assume what packages someone might already have installed as you can build this from any number of environments including a Fedora podman container. You should also add `vulkan-headers` and `zip` (will install `unzip` as a dependency) to the install list. That's odd, I just compiled Cemu without needing llvm. A recent PR must have fixed that. Ok, it looks like we don't need that any longer.

Never ever had to install llvm it worked without installing not sure what your doing but it’s not right and in is fine I won’t be changing it

Reply:
Do you actually read what I wrote?!  
No offense, but I don't think you're the right person for this job. 
I specifically stated ```it looks like we don't need that any longer```
As in you do not need to install ```llvm```
I don't know how you could have possibly misinterpret that. Just do whatever you're going to do. I'll correct your shortcomings later.

Reply:
> Pardon my interjection, but using `1.` (instead of `1)`) is the correct way to create numbered lists in Markdown.

1) works too

Reply:
@Tachi107 On **GitHub's** *custom* flavor of Markdown. It's not a part of the Markdown specification.